chainer/variable.py

```diff
@@ -67,15 +66,12 @@ class Variable(object):
     creator is called a *root* variable. A variable is root if it is created by
     the user, or if the reference is deleted by :meth:`unchain_backward`.
 
-    Users can disable this chaining behavior by setting the volatile flag for
-    the initial variables. When a function gets volatile variables as its
-    inputs, the output variables do not hold references to the function. This
-    acts like unchaining on every function application.
+    Users can disable this chaining behavior by calling
+    :func:`~chainer.no_backprop_mode` and :func:`~chainer.force_backprop_mode`.
+    In the previous context, a variable does not create a computational graph.
```

Comment:
How about the followings?

In the former context, a variable never creates a computational graph, whereas in the latter context, it is forced to create.